### PR TITLE
Upgraded the sample to use ASP.NET Core 2.0.

### PR DESCRIPTION
**Pull request by jaliyaudagedara**
Fellows, I have upgraded the sample project to use ASP.NET Core 2.0. Added a TODO comment, appreciate if you guys can suggest the best approach.

Thanks.

### CODE REVIEW

File: WebApp-OpenIDConnect-DotNet/Startup.cs [hunk 1]
```diff
@@ -35,19 +36,35 @@ public void ConfigureServices(IServiceCollection services)
...
+                .AddAuthentication(options =>
+                {
+                    options.DefaultAuthenticateScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                    options.DefaultSignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                    options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                })
+                .AddCookie()
+                .AddOpenIdConnect(options =>
+                {
+                    // TODO: Hack, use DI
```
Comment (jaliyaudagedara) [inline WebApp-OpenIDConnect-DotNet/Startup.cs]: Guys, how can I use injected IConfigureOptions<OpenIdConnectOptions> here?

**Comment by parakhj** [comment]
One of the branches has the code for v2.0 https://github.com/Azure-Samples/active-directory-b2c-dotnetcore-webapp/tree/core2.0

**Comment by jaliyaudagedara** [comment]
Oh my bad, sorry for that! Thanks anyway!

**Comment by parakhj** [comment]
No worries. Thanks for making the PR!

**Pull request closed without merging**